src/Components/benchmarkapps/Wasm.Performance/TestApp/Shared/FastGrid/Grid.razor

```diff
@@ -0,0 +1,48 @@
+﻿@typeparam TRowData
+
+<CascadingValue IsFixed="true" Value="this">@ChildContent</CascadingValue>
+
+<table @attributes="@Attributes">
+    <thead>
+        <tr>
+            @foreach (var col in columns)
+            {
+                col.RenderHeader(__builder);
```

Comment:
On second thought, would it make sense to expose these two method as RenderFragment properties in the column component, and call them with the `@` syntax here?

The reason I ask, is because while we can "cheat" to get the fastest render possible, we probably don't want to encourage customers to rely on things like `__builder`.

Thoughts?

Reply:
For the most optimized grid we can do, I'm happy to "cheat" in all possible ways that make a difference :)

We have a different scenario representing the kinds of code we expect 3rd parties to write (PlainTable).